styles.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  height: 100vh;
+  overflow: hidden;
```

Comment:
Let me know why you add that property. I would suggest overflow hidden property is useless here so it should be removed.